notebooks/dataproc/spark_rapids_ml.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+RAPIDS_VERSION=23.2.0
+
+# upgrade pip
+pip install --upgrade pip
+
+# patch existing
+pip install --ignore-installed "llvmlite<0.40,>=0.39.0dev0" "numba>=0.56.2"
+
+# install cudf and cuml
+pip install cudf-cu11==${RAPIDS_VERSION} \
+cuml-cu11==${RAPIDS_VERSION} \
+pylibraft-cu11==${RAPIDS_VERSION} \
+rmm-cu11==${RAPIDS_VERSION} \
+--extra-index-url=https://pypi.nvidia.com
```

Comment:
Heads up that for ucx in knn you will most likely still need to patch the pip install with the symlinks as in the Databricks init script.   If that doesn't work due to permissions, there are other ways to link within the site-packages.